spec/support/idp_template.rb

```diff
@@ -7,6 +7,8 @@
 route "get '/saml/auth' => 'saml_idp#new'"
 route "post '/saml/auth' => 'saml_idp#create'"
 route "get '/saml/logout' => 'saml_idp#logout'"
+route "get '/saml/sp_sign_out' => 'saml_idp#sp_sign_out'"
 
 template File.expand_path('../saml_idp_controller.rb.erb', __FILE__), 'app/controllers/saml_idp_controller.rb'
 copy_file File.expand_path('../saml_idp-saml_slo_post.html.erb', __FILE__), 'app/views/saml_idp/saml_slo_post.html.erb'
+copy_file File.expand_path('../application.css', __FILE__), 'public/stylesheets/application.css'
```

Comment:
:bow: Awesome idea. If you didn't want to create a new file in this project, you could also use `create_file` and just pass an empty string.